tests/regression/test_network_of_populations.py

```diff
@@ -38,7 +39,7 @@ def test_basic_simulation(data_api):
     )
 
     result = calculateInfectiousOverTime(
-        np.basicSimulationInternalAgeStructure(network, {"S08000016": {"[17,70)": 10.0}}),
+        np.basicSimulationInternalAgeStructure(network, {"S08000016": {"[17,70)": 10.0}}, numpy.random.default_rng(123)),
```

Comment:
Not really important, but I guess in this and the tests below we could pass None because they are all deterministic network tests

Reply:
That parameter is not optional. Maybe it should be, I don't know. I usually try to avoid optional parameters unless they are very convenient. In the case of basicSimulationInternalAgeStructure, I think using the stochastic method is the default and it's not obvious from the signature whether it's being used or not. So I think always requiring it is probably a good idea.